index.html

```diff
@@ -0,0 +1,533 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/5b1eeceadd.js" crossorigin="anonymous"></script>
+  <title>TNW-Website-Clone</title>
+  <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+  <header>
+    <nav class="nav">
```

Comment:
- [ ] In the original page in the small screen, the Navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to seach icon), but in your version, the search icon is missing and the hamburger bars are not aligned right. Please fix these issues.

![image](https://user-images.githubusercontent.com/60184961/93028827-301f5400-f61f-11ea-82f5-87e1d01b815f.png)